README.md

```diff
@@ -1,15 +1,40 @@
+[![example branch parameter](https://github.com/github/docs/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/ESMValGroup/ESMValTool/actions)
 [![Codacy Badge](https://app.codacy.com/project/badge/Coverage/79bf6932c2e844eea15d0fb1ed7e415c)](https://www.codacy.com/gh/ESMValGroup/ESMValTool?utm_source=github.com&utm_medium=referral&utm_content=ESMValGroup/ESMValTool&utm_campaign=Badge_Coverage)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/79bf6932c2e844eea15d0fb1ed7e415c)](https://www.codacy.com/gh/ESMValGroup/ESMValTool?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ESMValGroup/ESMValTool&amp;utm_campaign=Badge_Grade)
 [![Docker Build Status](https://img.shields.io/docker/cloud/build/esmvalgroup/esmvaltool.svg)](https://hub.docker.com/r/esmvalgroup/esmvaltool/)
 [![Anaconda-Server Badge](https://anaconda.org/esmvalgroup/esmvaltool/badges/installer/conda.svg)](https://conda.anaconda.org/esmvalgroup)
 
-ESMValTool: A community diagnostic and performance metrics tool for routine evaluation of Earth system models in CMIP
+![esmvaltoollogo](https://github.com/ESMValGroup/ESMValTool/blob/master/doc/sphinx/source/figures/ESMValTool-logo.png)
+
+- [**Documentation**](https://docs.esmvaltool.org/en/latest/)
+- [**ESMValTool Website**](https://www.esmvaltool.org/)
+- [**ESMValGroup Project on GitHub**](https://github.com/ESMValGroup)
+- [**Gallery**](https://docs.esmvaltool.org/en/latest/gallery.html)
+
+# Introduction
+
+ESMValTool is a community-developed climate model diagnostics and evaluation software package, driven
+both by computational performance and scientific accuracy and reproducibility. ESMValTool is open to both
+users and developers, encouraging open exchange of diagnostic source code and evaluation results from the
+Coupled Model Intercomparison Project [CMIP](https://www.wcrp-climate.org/wgcm-cmip) ensemble. For a
+comprehensive introduction to ESMValTool please visit our
+[documentation](https://docs.esmvaltool.org/en/latest/introduction.html) page.
+
+# Running esmvaltool
+
+Diagnostics from ESMValTool are run using [recipe](https://docs.esmvaltool.org/en/latest/recipes/index.html)
+files that contain pointers to the requested data types, directives for the preprocessing steps that data
+will be subject to, and directives for the actual diagnostics that will be run with the now preprocessed data.
+Data preprocessing is done via the [ESMValCore](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/quickstart/index.html) package, a pure Python, highly-optimized scientific library, developed by the ESMValTool core developers,
+and that performs a number of common analysis tasks
+such as regridding, masking, levels extraction etc. [Diagnostics](https://docs.esmvaltool.org/en/latest/develop/diagnostic.html) are written in a variety of programming languages (Python, NCL, R, julia) and are developed by the wider
```

Comment:
IMHO, Julia starts with the upper-case letter J, as it is written in its [documentation](https://docs.julialang.org/en/v1/) e.g. "Welcome to the documentation for Julia 1.5.". I might miss something here so please feel free to ignore my comment.